bundles/org.openhab.binding.bluetooth.grundfosalpha/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -50,6 +50,8 @@
 		<label>Grundfos Alpha Reader MI401</label>
 		<description>A Grundfos Alpha Reader MI401</description>
 		<category>Pump</category>
+		<semantic-equipment-tag>Pump</semantic-equipment-tag>
```

Comment:
I was literally just creating a PR for this binding when yours arrived. 😄 I'm not sure this is fully straight-forward. Technically, this is a Bluetooth device connected to the bump and providing readings for the pump. So the reader itself is not a pump.

In line 88 it can be safely added for the `alpha3` Thing, which is a pump with built-in Bluetooth.

Reply:
Correct. Do you think this would be worth clarifying in some documentation, perhaps https://next.openhab.org/docs/developer/bindings/semantic-tags.html#d-rules-for-extending-point-tags?

Will you add the tag also for the `alpha3` Thing?

Reply:
I did the alpha3 already.

Could you please remind me elsewhere regarding the docs?